packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -231,68 +231,7 @@ describe('HttpHandler', () => {
     }
   });
 
-  context('with a formData-parameter route', () => {
-    beforeEach(givenFormDataParamController);
-
-    it('returns the value sent in json-encoded body', () => {
-      return client
-        .post('/show-formdata')
-        .send({key: 'value'})
-        .expect(200, 'value');
-    });
-
-    it('rejects url-encoded request body', () => {
-      logErrorsExcept(415);
-      return client
-        .post('/show-formdata')
-        .send('key=value')
-        .expect(415);
-    });
-
-    it('returns 400 for malformed JSON body', () => {
-      logErrorsExcept(400);
-      return client
-        .post('/show-formdata')
-        .set('content-type', 'application/json')
-        .send('malformed-json')
-        .expect(400);
-    });
-
-    function givenFormDataParamController() {
-      const spec = anOpenApiSpec()
-        .withOperation('post', '/show-formdata', {
-          'x-operation-name': 'showFormData',
-          parameters: [
-            <ParameterObject>{
-              name: 'key',
-              in: 'formData',
-              description: 'Any value.',
-              required: true,
-              type: 'string',
-            },
-          ],
-          responses: {
-            200: {
-              schema: {
-                type: 'string',
-              },
-              description: '',
-            },
-          },
-        })
-        .build();
-
-      class RouteParamController {
-        async showFormData(key: string): Promise<string> {
-          return key;
-        }
-      }
-
-      givenControllerClass(RouteParamController, spec);
-    }
-  });
-
-  context('with a body-parameter route', () => {
+  context('with a body request route', () => {
```

Comment:
I presume there are tests coming here :)

Reply:
don't trust the git compare :( the old test cases are updated, if you are talking about the acceptance tests, add them in commit https://github.com/strongloop/loopback-next/pull/1067/commits/8e7570c12499304d4a2b28272da9db7b20ca60cc :)

Reply:
I think @b-admike might be referring to the `formdata` tests. I don't see them in the above commit. Does this have to do with how V3 handles the form data?

Reply:
@virkt25 V3 doesn't have a concept of "form data", it always treats `requestBody` as a whole, you can describe `requestBody` by any complicated schema, but you could not specify an argument as part of a `requestBody` schema. So I refactored all the V2 "form data" + "request body" tests to be V3 "request body" tests.